### PR TITLE
cli: add mechanism to ignore print events

### PR DESCRIPTION
**Pull request by lucasmoura**
## Proposed Commit Message
cli: add mechanism to ignore print events

When running some UA commands, we will be required to provide their
output in a machine-readable way. When that happens, we don't to print
anything to stdout. We are now adding a mechanism that allows us to
achieve that

PS: This is an alternative for #1904 

## Test Steps
Run the unittests and any integration test that is printing to stdout

## Desired commit type

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: uaclient/event_logger.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```
Comment (orndorffgrant) [inline uaclient/event_logger.py:1]: Do we need this line?

**Review by orndorffgrant — APPROVED** [review]
Looks good! I'll just run some of the tests jenkins didn't like to make sure they're okay

And one tiny comment

File: uaclient/event_logger.py (hunk 1 above)
Comment (lucasmoura) [reply uaclient/event_logger.py:1]: To be honest, I don't think we need it. I just followed the "template" of the `cli` module. But I agree we can remove it

**Comment by lucasmoura** [comment]
@orndorffgrant Thanks for the review on that PR, but I think we can use #1912 altogether. I have made some modifications already on it regarding the `print` part of the event logger. Sorry for that, I should have closed this PR earlier

**Comment by orndorffgrant** [comment]
@lucasmoura All good. I was aware but thought we were going to merge this first as a stepping stone and also to make the diff for #1912 a bit smaller. But this strategy works for me too :)

**Pull request closed without merging**